src/app/auth/handleRequest.js

```diff
@@ -15,16 +15,18 @@ function redirectResponse(location, code = 302, cookies) {
 async function handleRequest(cookies, queryStringParamCode, queryStringParamState, dynamoDBClient) {
     let session = new Session(cookies, dynamoDBClient);
     await session.init();
-
     if (session.sessionId === false) {
         return redirectResponse('/login');
     }
-    const state = session.state;
+    const state = session.getValue('state');
     const OIDC = new OpenIDConnect();
     try {
         const claims = await OIDC.authorize(queryStringParamCode, state, queryStringParamState, queryStringParamState);    
         if (claims) {
```

Comment:
is claims idd echt alleen waar als de oidc. authorize gelukt is? Of kan die ook waar zijn maar dan met een error code erin?

Reply:
Hij kan alleen de claims (met inhoud) teruggeven of hij throwt een javasccript error (die ik hierin opvang). In de claims horen geen errorcodes terug te komen (dan zou eerder in de authorize-stap al een error worden gegenereerd)